raspi/controlserver/controlserver.py

```diff
@@ -22,6 +23,18 @@
 wifi_search_folder = os.path.join(dir_path, '../access_point')
 susiconfig = '/home/pi/SUSI.AI/bin/susi-config'
 
+config_json = json_config.connect('/home/pi/SUSI.AI/config.json')
+
+def get_token():
+    url = 'http://api.susi.ai/aaa/login.json?type=access-token'
+    login = config_json['login_credentials']['email']
+    password = config_json['login_credentials']['password']
+    PARAMS = {
+        'login':login,
+        'password':password,
+    }
+    r1 = requests.get(url, params=PARAMS).json()
```

Comment:
If you call `requests` in the form of `requests.get(url, params)`, you should move `type=access-token` out of `url` also.

Reply:
Is this resolved? If yes, please mark it so.

Reply:
Same with the other commetns of @hongquan

Reply:
This is resolved, thanks